src/Views/Pages/Help/Help.js

```diff
@@ -24,12 +31,48 @@ const useStyles = makeStyles(theme => ({
 function Help() {
     document.title = "Help";
     const classes = useStyles(Theme);
+    const headers = ["LOGIN", "GAMES", "SETTINGS", "MISC"];
+
+    const output = (headerContent, bodyContent, index) => (
+        <div className={clsx(classes.pad)} key={index}>
+            <Typography color="secondary" variant="h2">
```

Comment:
wrap the typography and divider in one pad div

Reply:
Put 
<div className={clsx(classes.row>
  <div className={clsx(class.pad>
    <Typography...
<Divider/>
</div>
</div>  
<div className={clsx(classes.row)}>
  <div className={clsx(classes.pad)}>
    <Typography...
</Typography>
 </div>
</div>